packages/compass-connect/src/modules/telemetry.js

```diff
@@ -0,0 +1,94 @@
+import { createLoggerAndTelemetry } from '@mongodb-js/compass-logging';
+import { isAtlas, isLocalhost, isDigitalOcean } from 'mongodb-build-info';
+import { getCloudInfo } from 'mongodb-cloud-info';
+import ConnectionString from 'mongodb-connection-string-url';
+
+const { track, debug } = createLoggerAndTelemetry('COMPASS-CONNECT-UI');
+
+async function getConnectionData(connectionInfo) {
+  const {connectionOptions: {connectionString, sshTunnel}} = connectionInfo;
+  const connectionStringData = new ConnectionString(connectionString);
+  const hostName = connectionStringData.hosts[0];
+  const { isAws, isAzure, isGcp } = await getCloudInfo(hostName).catch((err) => {
+    debug('getCloudInfo failed', err);
+    return {};
+  });
+  const isPublicCloud = isAws || isAzure || isGcp;
+  const publicCloudName = isAws ? 'AWS' : isAzure ? 'Azure' : isGcp ? 'GCP' : '';
+  const authMechanism = connectionStringData.searchParams.get('authMechanism');
+  const authType = authMechanism
+    ? authMechanism
+    : connectionStringData.username
+      ? 'SCRAM-SHA-1'
```

Comment:
this should be 'DEFAULT' and 'DEFAULT' should be .. buh 'NONE'? or just '' i guess

```
const authType = authMechanism
    ? authMechanism
    : connectionStringData.username ? 'DEFAULT' : 'NONE'
```